saltproc/saltproc.py

```diff
@@ -163,37 +356,61 @@ def reopen_db(self, restart):
             if False, simply load the datasets
         """
         self.f = h5py.File(self.db_file, 'r+')
-        self.keff_db = self.f['keff_EOC']
-        self.keff_db_0 = self.f['keff_BOC']
-        self.bu_adens_db_0 = self.f['core adensity before reproc']
-        self.bu_adens_db_1 = self.f['core adensity after reproc']
-        self.tank_adens_db = self.f['tank adensity']
-        self.noble_adens_db = self.f['noble adensity']
-        self.th_adens_db = self.f['Th tank adensity']
-        self.isolib_db = self.f['iso codes']
-        self.number_of_isotopes = len(self.isolib_db)
-        self.keff = self.keff_db[0, :]
+        self.keff_eoc_db = self.f['keff_EOC']
+        self.keff_boc_db = self.f['keff_BOC']
+        self.driver_before_db = self.f['driver composition before reproc']
+        self.driver_refill_tank_db = self.f['driver refill tank composition']
+        self.driver_after_db = self.f['driver composition after reproc']
+
+        self.blanket_before_db = self.f['blanket composition before reproc']
```

Comment:
not all reactors have blankets!!! This stuff has to be removed and generalized. Try a reactor description class.